flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_AirportsMonitor.js

```diff
@@ -143,8 +150,6 @@ class CDUAirportsMonitor {
 
         // display data on MCDU
         if (this.icao1) {
-            mcdu.clearDisplay();
-            mcdu.page.Current = mcdu.page.AirportsMonitor;
```

Comment:
wondering why you removed the page id system, there are some things that do rely on it.

E.g. auto transition from init-b page to fuel pred when one engine is started.

Reply:
It is not necessary anymore. The auto-page switch is here: https://github.com/flybywiresim/a32nx/pull/5383/files#diff-064aac8a28c9cf6d37304e9f4dd8b05b4b09dcbb6e380edb1bcf06c60724eb1aR263